tests/integration/test_project.py

```diff
@@ -156,3 +165,14 @@ def test_can_open_a_shell_for_a_specific_container(self, mocked_call, persistent
         assert mocked_call.call_count == 1
         assert mocked_call.call_args[0][0] == \
             'lxc exec {} -- su -m root'.format(persistent_container.lxd_name)
+
+    def test_set_lxc_config(self):
```

Comment:
I think that you should've placed this test in `integration/test_container.py` because it is related to a feature implemented in the `Container` class. :wink: